Adyen/UI/Form/Items/Toggle/FormTogglechItem.swift

```diff
@@ -8,7 +8,10 @@ import Foundation
 
```

Comment:
@eduardourso Thanks so much for your contribution!

please make the pull request from develop branch, you seem to have made the pull request from master which is way behind develop, so we can't merge this into develop.